### PR TITLE
added PRNGKey function to jax.random

### PR DESCRIPTION
**Pull request by andyjaramillo**
#9407

### CODE REVIEW

File: ivy/functional/frontends/jax/random.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import ivy
```
Comment (jkeane508) [inline ivy/functional/frontends/jax/random.py:1]: we should add a comment above that this is a global import

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi,

Thanks for submitting this PR :)

Firstly, before we can merge we would need to undo the changes in the `.idea/` files as they cannot be changed on the main repo.

Secondly, the task requires that you write a unit test in the relevant file, however, due to the behavior of this function you would need to write a custom test that ensures that they are the same data type.

Finally, in the functional implementation we should only use Ivy functions to generate the keys :)

Good luck!

File: ivy/functional/frontends/jax/random.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    from jax._src.prng import PRNGKeyArray as _PRNGKeyArray
+except ImportError:
+    from types import SimpleNamespace
+
+    random = SimpleNamespace
+
+
+def PRNGKey(seed: int) -> ivy.Array:
+    ivy.seed(seed_value=seed)
+    return random().PRNG(seed)
```
[the remark this answers is not in the document]
Comment (andyjaramillo) [reply ivy/functional/frontends/jax/random.py:15]: Hi, thank you for responding to the post and I apologize for the late response. What ivy functions exactly are you referring to since I could not find them? Thank you

[the remark this answers is not in the document]
Comment (jkeane508) [reply ivy/functional/frontends/jax/random.py]: As we are generating random numbers we can use the ivy random functions which are documented [here](https://lets-unify.ai/ivy/functional/ivy/random.html)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
